src/components/EditSections/EditUserInfo/components/ExternalLinkModal/ExternalLinkModal.js

```diff
@@ -23,37 +23,41 @@ const ExternalLinkModal = ({
   const [inputValue, setInputValue] = useState('');
   const previousInputValue = useRef(profilePictureLink);
   const [disabled, setDisabled] = useState(false);
-  const [error, setError] = useState(false);
-  const externalURLValidityError = error ?
-    <FormattedMessage id="ui-users.information.profilePicture.externalLink.modal.externalURL.errorMessage" />
-    : null;
+  const [externalURLValidityError, setExternalURLValidityError] = useState(null);
 
   useEffect(() => {
     setInputValue(profilePictureLink);
-    setError(false);
   }, [profilePictureLink]);
 
   useEffect(() => {
     if (inputValue) {
       setDisabled(previousInputValue.current === inputValue);
-      setError(false);
-    } else {
-      setDisabled(true);
+      setExternalURLValidityError(null);
     }
   }, [inputValue]);
 
-  const handleSave = () => {
-    onSave(inputValue);
+  const handleSave = async () => {
+    const isValidImgURL = await isAValidImageUrl(inputValue);
+    if (isValidImgURL) {
+      onSave(inputValue);
+    }
```

Comment:
Image URL need not be validated once again as it will be already validated onBlur.
It can be saved in case of no error?